src/win/dl.c

```diff
@@ -64,7 +64,7 @@ void uv_dlclose(uv_lib_t* lib) {
 
 
 int uv_dlsym(uv_lib_t* lib, const char* name, void** ptr) {
-  *ptr = (void*) GetProcAddress(lib->handle, name);
+  *ptr = (void*)(uintptr_t) GetProcAddress(lib->handle, name); /* Cast though integer to suppress pedantic warning about forbidden cast. */
```

Comment:
I don't see MSVS complaining (I also don't think it should). so maybe leave it alone?

Reply:
GCC complains, since officially it's UB (although only under -Wpedantic). It's pretty silly, but that's what it is, and this is the only way to disable that warning that I'm aware of.

ref https://stackoverflow.com/questions/36645660/why-cant-i-cast-a-function-pointer-to-void

Reply:
Ack